docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx

```diff
@@ -32,114 +32,68 @@ This guide will talk through how to setup Teleport with Jira Server. Teleport to
+(!docs/pages/includes/plugins/identity-export.mdx!)
 
-Log into Teleport Authentication Server, this is where you normally run `tctl`. Create a
-new user and role that only has API access to the `access_request` API. The below script
-will create a yaml resource file for a new user and role.
-
-```bash
-cat > rscs.yaml <<EOF
-kind: user
-metadata:
-  name: access-plugin-jira
-spec:
-  roles: ['access-plugin-jira']
-version: v2
----
-kind: role
-metadata:
-  name: access-plugin-jira
-spec:
-  allow:
-    rules:
-      - resources: ['access_request']
-        verbs: ['list','read','update']
-    # teleport currently refuses to issue certs for a user with 0 logins,
-    # this restriction may be lifted in future versions.
-    logins: ['access-plugin-jira']
-version: v5
-EOF
-
-# ...
-$ tctl create -f rscs.yaml
-```
-
-#### Export access-plugin Certificate
-
-Teleport Plugin uses the `access-plugin-jira` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign).
-
-```code
-$ tctl auth sign --format=tls --user=access-plugin-jira --out=auth --ttl=2190h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuration-file).
-
-<Admonition
-  type="note"
-  title="Certificate Lifetime"
->
-  By default, [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign) produces certificates with a relatively short lifetime. For production deployments, the `--ttl` flag can be used to ensure a more practical certificate lifetime. `--ttl=8760h` exports a 1 year token
-</Admonition>
+We'll reference these files later when [configuring the plugins](#configuration-file).
 
 ### Setting up your Jira Server instance
 
 #### Creating a Project
 
-Teleport Jira Plugin relies on your Jira project having a board with at least three statuses (columns): Pending, Approved, and Denied. It's therefore the easiest scenario to create a new Jira project for Teleport to use.
+The Teleport Jira plugin relies on your Jira project having a board with at least three statuses (columns): Pending, Approved, and Denied. It's therefore the easiest scenario to create a new Jira project for Teleport to use.
 
-The specific type of project you choose when you create it doesn't matter, as long as you can setup a Kanban Board for it, but we recommend that you go with Kanban Software Development — this will reduce the amount of setup work you'll have to do and provide the board out of the box.
+The specific type of project you choose when you create it doesn't matter, as long as you can setup a Kanban Board for it, but we recommend that you go with Kanban Software Development—this will reduce the amount of setup work you'll have to do and provide the board out of the box.
```

Comment:
```suggestion
The specific type of project you choose when you create it doesn't matter, as long as you can setup a Kanban Board for it, but we recommend that you go with Kanban Software Development. This will reduce the amount of setup work you'll have to do and provide the board out of the box.
```